tests/jax/components/building/loggers_test.py

```diff
@@ -0,0 +1,157 @@
+import functools
+from typing import Callable
+
+import pytest
+
+from mava.components.jax.building.loggers import Logger, LoggerConfig
+from mava.core_jax import SystemBuilder
+from mava.systems.jax import Builder
+from mava.utils.loggers import logger_utils
+
+
+class TestLogger(Logger):
+    """Test Logger component."""
+
+    def __init__(
+        self,
+        test_logger_factory: Callable,
+    ):
+        """Create TestLogger
+
+        Args:
+            test_logger_factory: factory to use in the logger config.
+        """
+        logger_config = LoggerConfig()
+        logger_config.logger_factory = test_logger_factory
+        logger_config.logger_config = {
+            "trainer": {"time_stamp": "trainer_config"},
```

Comment:
Very minor, but can we make these `trainer_logger_config`, `executor_logger_config` and `evaluator_logger_config` instead? It makes reading this a littel bit easier.

Reply:
Agreed, done :)